docs/DeveloperGuide.md

```diff
@@ -722,14 +714,15 @@ The following activity diagram summarizes what happens when a user executes the
 #### Design considerations
 **Aspect: Marks assignment of a person in current displayed list as done or for any person in storage model:**
 
-* **Alternative 1:** (current choice) Marks assignment of a person in current displayed list
-    * Pros: Allows for a safer mark of assignments
-    * Cons: User has to carry out `list` command first if required person is not in the current displayed list
+* **Alternative 1:** (current choice) Marks assignment in current displayed assignment list.
+    * Pros: Allows for a safer mark of assignments.
+    * Cons: User has to carry out `list` command first if required person is not in the current displayed list, which is followed
+    by `show` command if assignments of required person are not shown.
 
-* **Alternative 2:** Marks assignment of any person with that name and assignment
-    * Pros: Allows user to mark assignment of a person without the need of additional commands
+* **Alternative 2:** Marks assignment of any person with that name and assignment.
```

Comment:
You can say "Marks an assignment as completed of the specified person, by in putting the name with the command